packages/popup-component/src/components/popup/popup.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('my-component', () => {
```

Comment:
We don't need this test isn't?

Reply:
No, but we can add a small render test using this file